server/views/pages/printNotification/printNotificationSlip.njk

```diff
@@ -0,0 +1,128 @@
+{% from "../partials/printNotification/sentenceTable.njk" import sentenceTable %}
+{% from "../partials/printNotification/adjustmentsTable.njk" import adjustmentsTable %}
+{% from "../partials/printNotification/keyDates.njk" import keyDates %}
+{% from "../partials/printNotification/prisonerDetails.njk" import prisonerDetails %}
+
+{% block header %}
+  {% if pageType == 'establishment' or pageType == 'offender' %}
+     <script data-qa="print-invoker" src="/assets/print.js"></script>
+  {% endif %}
+
+  <head>
+    <link href="/assets/stylesheets/application.css" media="all" rel="stylesheet" type="text/css" />
+  </head>
+
+  <body class="govuk-template__body  js-enabled" data-new-gr-c-s-check-loaded="14.1110.0" data-gr-ext-installed="">
+
+    <div class="print-width-container">
+      <div class="moj-width-container govuk-!-margin-bottom-4 govuk-!-display-none-print">
+        {% if fromPage == 'calculation' %}
+          <a data-qa="back-link" href="/calculation/{{ model.prisonerDetail.offenderNo }}/complete/{{ calculationRequestId }}">Back</a>
+        {% elif fromPage == 'view' %}
+          <a data-qa="back-link" href="/?prisonId={{ model.prisonerDetail.offenderNo }}">Back</a>
+        {% endif %}
+      </div>
+
+      <div class="moj-width-container govuk-!-margin-bottom-4">
+          <div class="govuk-grid-row print-button">
+            <div class="govuk-grid-column-full">
+              <h1 class="govuk-heading-l govuk-visually-hidden" >Print notification slip</h1>
+
+              <div class="govuk-button-group govuk-!-display-none-print">
+                {% if fromPage == 'calculation' %}
+                  {% set printLinks = '/calculation/' ~ model.prisonerDetail.offenderNo ~ '/summary/' ~ calculationRequestId ~ '/printNotificationSlip?fromPage=' ~ fromPage %}
```

Comment:
possibly untested

Reply:
Already covered in GET /view/:nomsId/calculation-summary/:calculationRequestId/printNotificationSlip?fromPage=view should generate page
and 
GET /calculation/:nomsId/summary/:calculationRequestId/printNotificationSlip?fromPage=calculation should generate page